addons/stock/report/report_stockpicking_operations.xml

```diff
@@ -121,14 +121,6 @@
                                         <td class="text-end">
                                             <span t-field="ml.quantity">3.00</span>
                                             <span t-field="ml.product_uom_id" groups="uom.group_uom">units</span>
-                                            <span t-if="ml.move_id.product_packaging_id">
```

Comment:
Same thing, I wonder if that information isn't still useful for warehouse workers